### DISCUSSION TITLE
Styling a checkbox

### DISCUSSION THREAD

**Discussion by ghost** [trussworks/react-uswds#2435] [Q&A]
I hope this is a stupid/easy question.  I want to add a style to a checkbox.  This is what I am doing but nothing happens:
                <Checkbox
                    id={myId}
                    label={label}
                    checked={f.checked ? true : false }
                    onChange={(e) => { console.log("placeholder"); }}
                    style={{marginTop: '5rem' }}
                />
What am I doing wrong??

Thank you for your help,
Cynthia

**Comment by werdnanoslen** [comment]
USWDS [hides](https://github.com/uswds/uswds/blob/develop/packages/usa-checkbox/src/styles/_usa-checkbox.scss#L7-L10) the actual input, which is what the `Checkbox`'s default props get [applied to](https://github.com/trussworks/react-uswds/blob/main/src/components/forms/Checkbox/Checkbox.tsx#L42). Instead, you could put that style in a class, and pass that via `className` on the Checkbox, which in turn applies it to the [containing `div`](https://github.com/trussworks/react-uswds/blob/main/src/components/forms/Checkbox/Checkbox.tsx#L35)
